travis_configure.sh

```diff
@@ -85,7 +111,8 @@ before_script()
 #
 # - configures and compiles the project
 # - runs all unittests with 'make check'
-# - generates deb or rpm package depending on the current build configuration
+# - for builds on linux deb or rpm package are generated depending on the
```

Comment:
What about ` - for builds on linux, a deb or rpm package is generated depending on the"?